projects/gloo/pkg/plugins/ratelimit/plugin_test.go

```diff
@@ -89,6 +95,50 @@ var _ = Describe("Plugin", func() {
 		}
 	})
 
+	Context("rate limit ordering", func() {
+		JustBeforeEach(func() {
+			timeout := time.Second
+			params.Snapshot.Upstreams = []*gloov1.Upstream{
+				{
+					Metadata: core.Metadata{
+						Name:      "extauth-upstream",
+						Namespace: "ns",
+					},
+				},
+			}
+			rlSettings.RateLimitBeforeAuth = true
+			initParams.Settings = &gloov1.Settings{
+				RatelimitServer: rlSettings,
+				Extauth: &extauthv1.Settings{
+					ExtauthzServerRef: &core.ResourceRef{
+						Name:      "extauth-upstream",
+						Namespace: "ns",
+					},
+					RequestTimeout: &timeout,
+				},
+			}
+			rlPlugin.Init(initParams)
+		})
+
+		It("should be ordered before ext auth", func() {
+			filters, err := rlPlugin.HttpFilters(params, nil)
+			Expect(err).NotTo(HaveOccurred(), "Should be able to build rate limit filters")
+			Expect(len(filters)).To(Equal(1), "Should only have created one custom filter")
```

Comment:
more idiomatic to do
```suggestion
			Expect(filters).To(HaveLen(1), "Should only have created one custom filter")
```